templates/lobbys/lobby.html

```diff
@@ -199,9 +237,13 @@ <h3> Prompts </h3>
                         <td>[[prompt.end]]</td>
 
                         <td><a v-bind:href="'/lobby/' + lobbyInfo.lobby_id + '/prompt/' + prompt.prompt_id">Results</a></td>
+                        <td v-if="editPrompts">
+                            <input type="checkbox" @click="selectPrompt(prompt.prompt_id)">
+                        </td>
                     </tr>
                 </tbody>
             </table>
+            <input v-if="editPrompts" href="javascript:;" @click="deletePrompts" class="btn btn-danger" value="Delete Selected">
```

Comment:
why is this an input instead of a normal button?

Reply:
tried it as a normal button, looked too HTML-y, making it input let the bootstrap styles apply. Modeled it after the Submit button on the same page